docs/user-guide/debugging-services.md

```diff
@@ -452,7 +472,7 @@ I0707 17:35:46.017061   30031 proxysocket.go:246] New UDP connection from 10.244
 If you see error messages about not being able to contact the master, you
 should double-check your `Node` configuration and installation steps.
 
-### Is kube-proxy writing iptables rules?
+### Is kube-proxy writing iptables rules
```

Comment:
When you fix the TOC generation it would be nice to add these question marks back.